custom-connectors/inQubaJourney/apiProperties.json

```diff
@@ -0,0 +1,46 @@
+{
+  "properties": {
+    "connectionParameters": {},
+    "iconBrandColor": "#ff9900",
+    "capabilities": [],
+    "policyTemplateInstances": [
+      {
+        "templateId": "dynamichosturl",
+        "title": "Set Token Endpoint Host URL",
+        "parameters": {
+          "x-ms-apimTemplateParameter.urlTemplate": "https://@headers('HostURL')",
+          "x-ms-apimTemplate-operationName": [
+            "AcquireAccessToken"
+          ]
+        }
+      },
+      {
+        "templateId": "dynamichosturl",
+        "title": "Set Host URL",
+        "parameters": {
+          "x-ms-apimTemplateParameter.urlTemplate": "https://@headers('TenantName').inqubacx.com",
+          "x-ms-apimTemplate-operationName": [
+            "PublishEvent",
+            "PublishTransaction"
+          ]
+        }
+      },
+      {
+        "templateId": "setheader",
+        "title": "Set Authorization Token Header",
+        "parameters": {
+          "x-ms-apimTemplateParameter.name": "Authorization",
+          "x-ms-apimTemplateParameter.value": "@headers('AuthorizationToken')",
+          "x-ms-apimTemplateParameter.existsAction": "override",
+          "x-ms-apimTemplate-policySection": "Request",
+          "x-ms-apimTemplate-operationName": [
+            "PublishEvent",
+            "PublishTransaction"
+          ]
+        }
+      }
+    ],
+    "publisher": "Prinay Panday",
+    "stackOwner": "Inquba Customer Intelligence Pty Ltd

```

Comment:
@prinayinq It is a syntax error, the stack owner property value should be double quoted.

Reply:
Syntax error fixed.